frontend/src/components/GroupView/RightView/index.tsx

```diff
@@ -7,24 +7,51 @@ import TaskCreator from '../../TaskCreator';
 import { promptTextInput } from '../../Util/Modals';
 import { updateGroup } from '../../../firebase/actions';
 
+type State = {
+  readonly taskCreatorOpened: boolean;
+  readonly assignedMember?: SamwiseUserProfile;
+};
+
 type Props = {
   readonly group: Group;
-  readonly groupMemberProfiles: readonly SamwiseUserProfile[];
+  readonly groupMemberProfiles: SamwiseUserProfile[];
   readonly tasks: readonly Task[];
 };
 
 const RightView = ({ group, groupMemberProfiles, tasks }: Props): ReactElement => {
+  const [{ taskCreatorOpened, assignedMember }, setState] = useState<State>({
```

Comment:
Maybe make a more descriptive name for this hook instead of `setState`, and even better maybe we can have a type for this data so we can pass a parameter into the generic here.